gwdetchar/omega/html.py

```diff
@@ -842,4 +842,6 @@ def write_about_page(configfiles):
         with open(configfile, 'r') as fobj:
             inifile = fobj.read()
         page.add(htmlio.render_code(inifile, 'ini'))
+    # add environment listing
+    page.add(str(htmlio.package_table()))
```

Comment:
@duncanmmacleod, apropos the comment above, this line can be done more simply:

```suggestion
    page.add(htmlio.package_table())
```

Reply:
As discussed privately, I have not changed this, since this way we don't rely on the upstream function returning a `str` explicitly. If we decide to change the return upstream in the future, we don't have to change this again.